src/common/Surface.js

```diff
@@ -216,7 +221,35 @@
             ;
         }
 
-        this._menu.element().node().parentNode.appendChild(this._menu.element().node());
+        //this._menu.element().node().parentNode.appendChild(this._menu.element().node()); WHAT DOES THIS DO?
```

Comment:
It ensure its "on top" from a SVG z-order point of view (you can add a comment for that!)